hammers/query.py

```diff
@@ -523,6 +523,33 @@ def blazar_set_non_reservable(db, node):
     '''
     return db.query(sql, args=[node], no_rows=True)
 
+@query
+def blazar_find_old_host_alloc(db):
+    """Find computehost allocations tied to expired leases"""
+    sql = '''\
+    SELECT ca.id, l.id AS lid, ch.hypervisor_hostname
+    FROM blazar.computehost_allocations ca
+    JOIN blazar.reservations r
+    ON ca.reservation_id=r.id
+    JOIN blazar.leases l
+    ON r.lease_id=l.id
+    JOIN blazar.computehosts ch
+    ON ca.compute_host_id=ch.id
+    WHERE ca.deleted is Null
+       AND l.end_date < curdate() - interval 7 day
```

Comment:
Why the extra buffer of one week? I'm not opposed to it, just wondering the reasoning

Reply:
I thought in case we want to recover a lease but that doesn't really make sense, I'll remove it.